### DISCUSSION TITLE
文档和教程

### DISCUSSION THREAD

**Discussion by YueYueZY** [halo-dev/halo#5566] [General]
### 你当前使用的版本

2.13.1

### 描述一下此特性

建议改进网站的文档和教程，以帮助用户更好地了解和使用网站提供的功能和服务。

**Comment by ruibaby** [comment]
此问题与此仓库本身无关，如果你对文档有具体的建议，请向 https://github.com/halo-dev/docs 提交 issue。不要提交笼统的改进建议。

/triage unresolved
